build.gradle

```diff
@@ -6,14 +6,16 @@ buildscript {
 
 plugins {
 	id "java"
+	id "java-library"
```

Comment:
```suggestion
	id "java-library"
```
`java-library` applies `java` (and I think Loom does too)

Reply:
Loom is already using `java`, yes.

Reply:
Java library is where you'll get the api configuration